### ISSUE TITLE
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

### ISSUE THREAD

**Issue by msftedad** [twbs/bootstrap#42031]
### Prerequisites

- [x] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7653)
Browser: Edge (Version 144.0.3719.82)
Tool: Color Contrast Analyzer Tool
URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

**User Impact:**
Users with low vision may struggle to identify which calendar date is focused because the focus indicator has insufficient contrast (1.2:1 instead of the required 3:1). This makes keyboard navigation and focus tracking difficult, reducing accessibility and usability.

**Repro Steps:**

1. Open the URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ in edge browser in windows.
2. Press tab key to navigate to the 'Select Date' control.
3. Calendar will open. 
4. Press tab key to navigate to the calendar dates. 
5. Open Color Contrast Analyzer Tool.
6. Verify whether Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 or not.

**Actual Result:**
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

**Refer Attachment:**

1. Luminosity contrast ratio of focus indicator on calendar dates is less than expected ratio.png
2. Luminosity contrast ratio of focus indicator on calendar dates is less than expected ratio.mp4

https://github.com/user-attachments/assets/da2b5fbe-5f55-49e3-91bb-842aa2b3c7a2
<img width="969" height="563" alt="Image" src="https://github.com/user-attachments/assets/f70dd602-5db3-448d-9b0a-176cb92dc6cc" />

**Expected Result:**
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

### Reduced test cases

https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Bootstrap are you using?

1.10.0

**Comment by julien-deramond** [comment]
Closing for the same reason as https://github.com/twbs/bootstrap/issues/42032#issuecomment-3776673670
